### PR TITLE
feat(PropTypes): Add PropTypes definitions

### PR DESCRIPTION
**Pull request by CHBresser**
Added PropTypes w/ Default Props where needed.

### CODE REVIEW

File: src/component/color-edition-list-item.js [hunk 1]
```diff
@@ -74,4 +75,14 @@ class ColorEditionListItem extends React.Component {
   }
 }
 
+ColorEditionListItem.propTypes = {
```
Comment (jdrouet) [inline src/component/color-edition-list-item.js]: Considering that this component is a class, the `propTypes` and `defaultProps` should probably be set as a static class property

File: src/component/color-bubble.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import cn from 'classnames';
 import { withStyles } from '@material-ui/core/styles';
 import Avatar from '@material-ui/core/Avatar';
+import PropTypes from "prop-types";
```
Comment (jdrouet) [inline src/component/color-bubble.js]: Could you change all the double quotes by single quotes?

File: src/component/color-bubble.js [hunk 3]
```diff
@@ -27,4 +28,9 @@ class ColorBubble extends React.PureComponent {
   }
 }
 
+ColorBubble.propTypes = {
```
Comment (jdrouet) [inline src/component/color-bubble.js]: Same than for `ColorEditionListItem`

**Comment by CHBresser** [comment]
First PR let me know if I need to do anything else. 

Edit: That was an accidental close.

**Review by jdrouet — CHANGES_REQUESTED** [review]
Thank you for your contribution. I really appreciate it. But I'd some improvements to your PR.
- Putting the `propTypes` and `defaultProps` as static class properties
- Removing the double quotes
- Merge your 2 commits

File: src/component/color-edition-list-item.js (hunk 1 above)
Comment (CHBresser) [reply src/component/color-edition-list-item.js]: Done.

File: src/component/color-bubble.js (hunk 2 above)
Comment (CHBresser) [reply src/component/color-bubble.js]: Done.

File: src/component/color-bubble.js (hunk 3 above)
Comment (CHBresser) [reply src/component/color-bubble.js]: Done

**Comment by CHBresser** [comment]
I am unaware of how to merge my commits. All the results on google seem to imply that you squash the commits when you merge the PR.

**Comment by CHBresser** [comment]
Not sure what the conflict is now on package.json

**Comment by jdrouet** [comment]
@CHBresser the procedure is the following.
- First move your commits to a different branch, it will be easier to pull our repo.
- then synchronise your repo with ours
- then rebase your branch to master
- then delete your last commit that might be useless due to the rebase
- squash the commits refering to the prop types

Also, I added the a linter on the project, so please take care that everything is ok, otherwise circleci won't let us merge your PR

**Pull request closed without merging**